support/cas-server-support-redis-ticket-registry/src/test/java/org/apereo/cas/ticket/registry/RedisServerTicketRegistryTests.java

```diff
@@ -35,7 +35,11 @@
 @TestPropertySource(properties = {
     "cas.ticket.registry.redis.host=localhost",
     "cas.ticket.registry.redis.port=6379",
-    "cas.ticket.registry.redis.pool.max-active=20"
+    "cas.ticket.registry.redis.pool.max-active=20",
+    "cas.ticket.registry.redis.sentinel.master=mymaster",
+    "cas.ticket.registry.redis.sentinel.node[0]=localhost:26379",
+    "cas.ticket.registry.redis.sentinel.node[1]=localhost:26380",
+    "cas.ticket.registry.redis.sentinel.node[2]=localhost:26381"
 })
```

Comment:
You should make sure this test is tagged with `@EnabledIfContinuousIntegration`, so it can run via CI. Right now, it only is set to run locally.

Reply:
So I didn't get how are running different test :cry: 
If i change this class with the tag that you provide, it will run as usual ?

Also could you provide the command to run test locally ?
I tried `../../gradlew --build-cache -DtestCategoryType=Redis test --parallel -x check --debug` from the package and from the project root path but I'm getting errors

Reply:
> If i change this class with the tag that you provide, it will run as usual ?

It will run under CI. Not locally. It only enables the test if it's running under continuous integration, as the annotation suggests.

> Also could you provide the command to run test locally ?

Please see https://apereo.github.io/2019/03/12/cas61x-codebase-feature-build/#running-tests

When you run the gradle command locally, you either have to pass an environment variable or system property that sets `CI` to true to bypass the rule imposed by `EnabledIfContinuousIntegration`, or you have to remove that annotation temporarily to let Gradle run it (This is what I usually do).

Reply:
@tduehr If you ave the time, it would be very useful to perhaps update the build process doc to explain how tests are selected and executed.

Reply:
@mmoayyed thanks for the explanations, at least locally I can run now the test and I reproduce the error, but how to set system properties to set a local CI env ? I ran ` ./ci/tests/redis/run-tests-redis.sh` but seems doesn't work, so ?

Reply:
You will need to set an environment variable called CI to true, before running the test:

```java
@Target({ METHOD, TYPE, ANNOTATION_TYPE })
@Retention(RUNTIME)
@EnabledIfEnvironmentVariable(named = "CI", matches = "true") // <--- See here
public @interface EnabledIfContinuousIntegration {
}
```

Reply:
Will do. It needs some clean up for consistency anyway. Mind if I do away with that class/test condition?

Reply:
@jgribonvald Definitely do not worry about clogging CI or anything like that. Take a look at the build history for PRs... It's a mess. If there's something you want to kill off or anything like that just hit us up in gitter or whereever...

Reply:
> Mind if I do away with that class/test condition?

Not at all. Have at it.

Reply:
So with @tduehr we've found incorrect configuration on redist test and so I've made an other commit to enable/disable redis test on CI or on local from the rigth context.